client/src/components/PersonCard/personcard.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PersonCard from './personcard';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('PersonCard', module)
+  .add("Default", () => <PersonCard
+  PersonCard__ProfileName="Lindsay L." 
+  PersonCard__Location="Washington D.C." 
+  PersonCard__Interests="#budgeting #technology #outdoors #traveling"
```

Comment:
Can we make these props names shorter? i.e. `name` and `location`? I think `interests` should maybe be an array

Reply:
Also, one thing to think about is we don't know if those hashtags are supposed to be clickable (eventually) or not!